.github/workflows/triage_handle_new_comments.yml

```diff
@@ -0,0 +1,33 @@
+name: 'Handle Issue/PR Comment Workflow'
+on:
+  # makes this workflow reusable
+  workflow_call:
+    secrets:
+      ADD_TO_TRIAGE_BOARD_TOKEN:
+        required: true
+
+  issue_comment:
+    types: [created]
+
+jobs:
+  handle-comment-scenarios:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          repository: 'cypress-io/release-automations'
+          ref: 'master'
+          ssh-key: ${{ secrets.WORKFLOW_DEPLOY_KEY }} 
```

Comment:
Does this secret also need to be included in the workflow call?

Reply:
no this secret is needed to checkout a private repo.